### PR TITLE
Fail on empty src directories.

### PR DESCRIPTION
**Pull request by runningcode**
Also add test for failing on empty src directories.

### CODE REVIEW

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ class PluginIntegrationTest constructor(private val version: String) {
         @JvmStatic
         @Parameterized.Parameters(name = "{0}")
         fun getParams(): List<String> {
-            return listOf("5.0", "5.3", "5.6", "6.0.1", "6.1.1")
+            return listOf("5.0", "5.1", "5.6", "6.0.1", "6.1.1")
```
Comment (emartynov) [inline doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: I see this only for these specific tests. Do you want also run other tests against different Gradle versions?

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt [hunk 2]
```diff
@@ -132,7 +132,7 @@ class PluginIntegrationTest constructor(private val version: String) {
                 google()
               }
               dependencies {
-                classpath("com.android.tools.build:gradle:3.4.2")
+                classpath("com.android.tools.build:gradle:3.5.3")
```
Comment (emartynov) [inline doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: Should it be some constant - so you don't need to update it in all tests files?

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt [hunk 3]
```diff
@@ -187,6 +187,32 @@ class PluginIntegrationTest constructor(private val version: String) {
...
+    fun testFailOnEmptyDirectories() {
+        assumeSupportedVersion()
+        writeBuildGradle(
+            """
+                    |plugins {
+                    |  id "com.osacky.doctor"
+                    |}
+                    |doctor {
+                    |  disallowMultipleDaemons = false
+                    |  ensureJavaHomeMatches = false
```
Comment (emartynov) [inline doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: Do you want to explicit mention here empty folders check?

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt (hunk 1 above)
Comment (runningcode) [reply doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: yeah, it just adds a lot of time and i keep getting ooms in the tests so i wont do it for now.

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt (hunk 2 above)
Comment (runningcode) [reply doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: very true, but its only used here for now.

File: doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt (hunk 3 above)
Comment (runningcode) [reply doctor-plugin/src/test/java/com/osacky/doctor/PluginIntegrationTest.kt]: added!
